openedx_tagging/core/tagging/urls.py

```diff
@@ -0,0 +1,10 @@
+"""
+Tagging API URLs.
+"""
+
+from django.urls import path, include
+
+from .rest_api import urls
+
+app_name = "oel_tagging"
```

Comment:
Removing this breaks the tests. Is there a way to set the app_name only for tests?

Reply:
Nah, it's ok to leave this here.